server.js

```diff
@@ -0,0 +1,157 @@
+"use strict";
+
+const express = require("express");
+const favicon = require("serve-favicon");
+const bodyParser = require("body-parser");
+const session = require("express-session");
+// const csrf = require('csurf');
+const consolidate = require("consolidate"); // Templating library adapter for Express
+const swig = require("swig");
+// const helmet = require("helmet");
+const MongoClient = require("mongodb").MongoClient; // Driver for connecting to MongoDB
+const http = require("http");
+const marked = require("marked");
+//const nosniff = require('dont-sniff-mimetype');
+const app = express(); // Web framework to handle routing requests
+const routes = require("./app/routes");
+const { port, db, cookieSecret } = require("./config/config"); // Application config properties
+/*
+// Fix for A6-Sensitive Data Exposure
+// Load keys for establishing secure HTTPS connection
+const fs = require("fs");
+const https = require("https");
+const path = require("path");
+const httpsOptions = {
+    key: fs.readFileSync(path.resolve(__dirname, "./artifacts/cert/server.key")),
+    cert: fs.readFileSync(path.resolve(__dirname, "./artifacts/cert/server.crt"))
+};
+*/
+
+MongoClient.connect(db, (err, db) => {
+    if (err) {
+        console.log("Error: DB: connect");
+        console.log(err);
+        process.exit(1);
+    }
+    console.log(`Connected to the database`);
+
+    /*
+    // Fix for A5 - Security MisConfig
+    // TODO: Review the rest of helmet options, like "xssFilter"
+    // Remove default x-powered-by response header
+    app.disable("x-powered-by");
+
+    // Prevent opening page in frame or iframe to protect from clickjacking
+    app.use(helmet.frameguard()); //xframe deprecated
+
+    // Prevents browser from caching and storing page
+    app.use(helmet.noCache());
+
+    // Allow loading resources only from white-listed domains
+    app.use(helmet.contentSecurityPolicy()); //csp deprecated
+
+    // Allow communication only on HTTPS
+    app.use(helmet.hsts());
+
+    // TODO: Add another vuln: https://github.com/helmetjs/helmet/issues/26
+    // Enable XSS filter in IE (On by default)
+    // app.use(helmet.iexss());
+    // Now it should be used in hit way, but the README alerts that could be
+    // dangerous, like specified in the issue.
+    // app.use(helmet.xssFilter({ setOnOldIE: true }));
+
+    // Forces browser to only use the Content-Type set in the response header instead of sniffing or guessing it
+    app.use(nosniff());
+    */
+
+    // Adding/ remove HTTP Headers for security
+    app.use(favicon(__dirname + "/app/assets/favicon.ico"));
+
+    // Express middleware to populate "req.body" so we can access POST variables
+    app.use(bodyParser.json());
+    app.use(bodyParser.urlencoded({
+        // Mandatory in Express v4
+        extended: false
+    }));
+
+    // Enable session management using express middleware
+    app.use(session({
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- c7de6e8152f44e4f65f866df3ef5d39b -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Secure Cookie Attribute Not Set
_Low CWE-614_
The configuration of the `client-sessions`, `cookie-session` or `express-session` library does not set the `secure` attribute of the session cookie. While web browsers send cookies with the `secure` attribute only over HTTPS, they do send cookies without it over HTTP. If an attacker can fool the victim's browser into sending an HTTP request to the vulnerable site (for example, with JavaScript code on a malicious page or with a phishing link), the victim's browser will send the vulnerable cookie in clear text. If the attacker can also intercept this network traffic, then they can steal the victim's session cookie and impersonate the victim.



### How to fix
Set the 'Secure' attribute should on all cookies that contain sensitive data, especially those containing session IDs. The mitigation details depend on the application server, with some requiring programmatic configuration, where others can be configured through an administration interface. 

If not already done, consider deploying HTTP Strict Transport Security (HSTS) headers to minimize the risk of plain-text HTTP requests to the application server.